daniellamarr/warren-web
# Make changes to the team section

#### What does this PR do?
- Currently, on the team section, the CTO is Daniel. This PR changes the CTO details.
#### Description of Task to be completed?
- Change CTO form Daniel to Osas
- Change CTO image
#### How should this be manually tested?
- Pull this branch `make-changes-to-team-section`
- Navigate to the root `index.html` file
- Scroll down to the team section to see changes
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-04-27 at 9 30 53 AM" src="https://user-images.githubusercontent.com/42177767/56847128-2df41280-68cf-11e9-8285-f609ef9e58b8.png">
